CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 0.18.0 - 2020-07-10
+### Added
+- Added support for Free App Service Trial Apps! Visit [here](https://code.visualstudio.com/tryappservice/) to Create and Import your own Free NodeJS Trial App.
```

Comment:
I assume Matt would want the telemetry endpoint thing here as well

Reply:
Yep, it's added! Also, please change "NodeJS" to "Node.js" and I'm good.